### DISCUSSION TITLE
CSP/DB/Storage targets and progress

### DISCUSSION THREAD

**Discussion by yunkon-kim** [cloud-barista/mc-data-manager#41] [Show and tell]
Hi, forks 👋 

This document shares and tracks the CSP/DB/Storage targets and progress for mc-data-manager.

We’ll be continuously updating the CSP/DB/Storage targets under consideration for "data backup and restore" and "data migration" tasks.

For more details, please refer to the link below.
:pushpin: https://docs.google.com/spreadsheets/d/1DmzCijltb4rUYzLbzYtfbWuenKBNmIA7ME-dWMsZd70/edit?usp=sharing

**Comment by yunkon-kim** [comment]
@heedaeshin 현황을 업데이트하여 주시기 바랍니다.

**Reply by heedaeshin** [reply]
@yunkon-kim 
금일중에 PR 진행되며 
해당 PR 관련 항목 정리될 예정입니다.

**Reply by heedaeshin** [reply]
https://github.com/cloud-barista/mc-data-manager/pull/48#issue-2509735768

**Reply by yunkon-kim** [reply]
@heedaeshin 
Live 문서도 함께 수정하여 주시기 바랍니다. 수정 권한이 필요하신 경우, 링크 접속 후 권한 요청해 주십시오.
